perception/lidar_apollo_segmentation_tvm/CMakeLists.txt

```diff
@@ -69,14 +69,6 @@ if(NOT NN_DEPENDENCY STREQUAL "")
     ${tvm_runtime_LIBRARIES}
   )
 
-  if(BUILD_TESTING)
```

Comment:
I don't think removing the tests is a good change.

I think a better behavior would be to keep what is in this file as is, and instead modify the test so that it checks whether the artifact files are present or not at the very begging; if there are not present print a warning (or something) and pass the test, and if they are present proceed with the test.

Reply:
Thank you for your feedback. I did some updates to address it. Could you please review it again?